WORKSPACE

```diff
@@ -127,3 +127,12 @@ http_file(
         "https://github.com/bazelbuild/bazel-toolchains/archive/44200e0c026d86c53470d107b3697a3e46469c43.tar.gz",
     ],
 )
+
+load("//rules:gsutil.bzl", "gsutil_cp")
+
+gsutil_cp(
+    name = "ubuntu16_04_debian_pkgs",
+    file = "ubuntu16-04-debian_pkgs.tar",
+    gcs_bucket = "gs://rbe-toolchain-debs",
```

Comment:
Let's avoid exposing these info in the external repo, if we could.

Reply:
Ok, sorry about it.